extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -62,6 +63,15 @@ export class PublishOptionsDialog {
 
 	private initializeDeploymentOptionsDialogTab(): void {
 		this.optionsTab?.registerContent(async view => {
+			// creating loader component
+			this.loader = view.modelBuilder.loadingComponent()
+				.withProps({
+					CSSStyles: {
+						'margin-top': '350px'
```

Comment:
how does this look if the ADS window is half the height as in the video? Instead of a hard coded pixel value, can this be set to the middle of the dialog a different way?

Reply:
I tried using the verticalalign and other properties, but since the div element of the loadingcomponent is set to top of the view and only adjusting margin is working best here! will see if there is any other possible way to do this. Thanks

Reply:
How about just putting a percentage for margin-top (or top)? Does this work: `'margin-top': '50%'`?

Reply:
still the same, the loader sticks at some height and changing the ADS window height cuts of the loader after some height:
![image](https://user-images.githubusercontent.com/74571829/184423251-4ca10e69-2f14-4b9a-a9ba-ae206ccff142.png)

Is there a way to get access the element from the tab using the class, so that if i can have the height of the tab, we can simply set the height/2 to the loader margin. like document.getElementByClassName() but here document is not defined
![image](https://user-images.githubusercontent.com/74571829/184423547-1a580129-6069-44df-9abf-ba66316a3b37.png)